packages/dawn4py/package.py

```diff
@@ -24,6 +24,8 @@ class Dawn4py(PythonPackage):
     depends_on('cmake', type='build')
     depends_on('py-setuptools', type='build')
     depends_on('py-protobuf', type=('build','run'))
+    depends_on('py-pytest', type=('build','run'))
+    depends_on('py-black', type=('build','run'))
```

Comment:
```suggestion
    depends_on('py-attr', type=('build','run'))
    depends_on('py-pytest', type=('build','run'))
    depends_on('py-black', type=('build','run'))
```

I think we also need `py-attr`:
https://github.com/MeteoSwiss-APN/dawn/blob/56b9695096c4278c7cdffa827deed348778795e8/dawn/setup.py#L49

We could also specify the version ranges, but I'm not sure whether they're accurate in the `setup.py` and I think that would scope creep this PR.